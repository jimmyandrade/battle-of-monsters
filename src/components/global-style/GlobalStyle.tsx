import type { FC } from 'react'
import type { GlobalStyleProps } from './GlobalStyle.types'
import { GlobalStyleRoot } from './GlobalStyle.styles'

export const GlobalStyle: FC<GlobalStyleProps> = (
  {
    children
  }
) => (
  <>
    <GlobalStyleRoot />
    {children}
  </>
)
