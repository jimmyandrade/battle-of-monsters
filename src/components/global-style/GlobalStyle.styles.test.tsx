import { GlobalStyleRoot } from './GlobalStyle.styles'
import { render } from '@testing-library/react'

describe('Global styles', () => {
  describe('Root component', () => {
    it('should match snapshot', () => {
      const { container } = render(<GlobalStyleRoot />)
      expect(container).toMatchSnapshot()
    })
  })
})
