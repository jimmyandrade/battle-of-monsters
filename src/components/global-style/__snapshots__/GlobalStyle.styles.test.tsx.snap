// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global styles Root component should match snapshot 1`] = `<div />`;
